app/renderer/components/tabs/tab.js

```diff
@@ -380,59 +401,56 @@ const styles = StyleSheet.create({
     width: globalStyles.spacing.sentinelSize
   },
 
-  tabId: {
-    justifyContent: 'center',
+  tab_audioTopBorder: {
```

Comment:
would you mind moving this under `tab_active`?

Reply:
Addressed on https://github.com/brave/browser-laptop/pull/10691/files#diff-4f164dab3a38a1443d7ba3288fef641eR408

Confirmed on https://github.com/brave/browser-laptop/blame/83864a8769ee10cb2d2d864b19289ba5e229bd34/app/renderer/components/tabs/tab.js#L408